frontend/src/containers/RecipePage/Recipe.js

```diff
@@ -0,0 +1,135 @@
+        );
+      }
+      return ""; 
+    };
+
+    return (
+      <div className="recipe-overall-container">
+        <div className="row">
+          <div className="header">
+            <Header />
+          </div>
+        </div>
+        <div>
+          <button className="btn navbar-btn btn-lg editButton" type="button">
+            <img src="https://img.icons8.com/windows/32/ffa616/edit.png" alt="edit"/>
+             <p id="edit-icon-text">Edit</p>
+          </button>
+          <button  className="btn navbar-btn btn-lg cancelButton" type="button">
+             <img src="https://img.icons8.com/windows/32/ffa616/cancel.png" alt="delete"/>
+              <p id="delete-icon-text">Delete</p>
+          </button>
+        </div>
+        <img src={recipephoto} alt="recipephoto" className="recipe-photo" />
+        <div className="recipe-specs-bars">
+          <div className="row">
+            <div id="recipe-spec-container1" className="recipe-spec-element col-sm-4 text-center">
+               <span> 
+                 {mealDifficulty()}
+                 {difficulty}
+               </span>
+             </div>
+            <div id="recipe-spec-container2" className="recipe-spec-element col-sm-4 text-center">
+              <span>
+                <i className="fas fa-clock" />
+                  { duration }{' '}MIN
+              </span>
+            </div>
+            <div id="recipe-spec-container3" className="recipe-spec-element col-sm-4 text-center">
+              <i className="fas fa-utensil-spoon" />
+                <span>{ servings }</span>
+            </div>
+          </div>
+        </div>
+        <div className="tags-bar">
+          <TagsBar/>
+        </div>
+        <div className="row" id="blurb-title">
+          <h1 id="recipe-title" className="col-sm-6 offset-sm-3">
+             {title}
+          </h1>
+          <p id="recipe-descrip" className="col-sm-8 offset-sm-2">
+            {snippet}
+          </p>
+        </div>
+        <div id="hr" />
+        <div className="separator col-sm-10 offset-sm-1" />
+        <div id="ingredient-box">
+          <div id="ingredient-list">
+            <h1 id="ingredient-title">Ingredients</h1>
+            <ul id="ingredient-ul">
+              {ingredient.map(ingred => (
+                <li>
+                  <div id="single-ingredient">{ingred}</div>
+                </li>
+              ))}
+            </ul>
+          </div>
+        </div>
+        <div id="hr" />
+        <div className="separator col-sm-10 offset-sm-1" />
+          <div id="direction-container" className="col-sm-10 offset-sm-1">
+            <h1 id="direction-title">Directions</h1>
+             <ul className="direction-list">
+                {directions.map((direction, index) => (
+                <div className="single-direction">
+                  <ol className="direction-index">{index + 1}</ol>
```

Comment:
Sorry, I guess I may have given a confusing suggestion here earlier. If you just want to use a `span` for this, that'd be better, I think. `ol` for a list item is a little odd.

I believe my previous suggestion was to make the whole direction list an `ol`, and have each item be an `li`.